openassessment/xblock/test/data/parse_rubric_xml.json

```diff
@@ -2,7 +2,6 @@
     "simple": {
         "xml": [
             "<rubric>",
-            "<prompt>Test prompt</prompt>",
```

Comment:
Actually in the xml the `prompt` is currently serialized inside the `rubric` element. I have moved it out to the top level. For example, here is a subset of the xml for the ORA problem in the demo course:
```
  <openassessment url_name="b24c33ea35954c7889e1d2944d3fe397" submission_start="2001-01-01T00:00:00+00:00" submission_due="2029-01-01T00:00:00+00:00" allow_file_upload="False" allow_latex="False">
    <title></title>
    <assessments>
      <assessment name="student-training"></assessment>
      <assessment name="peer-assessment" must_grade="1" must_be_graded_by="1" start="2001-01-01T00:00:00+00:00" due="2029-01-01T00:00:00+00:00"/>
      <assessment name="self-assessment" start="2001-01-01T00:00:00+00:00" due="2029-01-01T00:00:00+00:00"/>
    </assessments>
    <rubric>
      <prompt>    Describe your favorite meal, and write a persuasive argument to convince someone to try that meal. Support your position with facts and examples from your experience. 
</prompt>
      <criterion feedback="optional">
        <name>Ideas</name>
        <label>Ideas</label>
        <prompt>Determine if there is a unifying theme or main idea.</prompt>
      </criterion>
      <feedbackprompt>How does this meal compare with the meal you submitted?</feedbackprompt>
      <feedback_default_text>I noticed that this response...
</feedback_default_text>
    </rubric>
  </openassessment>
```